modules/bcftools/merge/main.nf

```diff
@@ -8,23 +8,38 @@ process BCFTOOLS_MERGE {
         'quay.io/biocontainers/bcftools:1.15.1--h0ea216a_0' }"
 
     input:
-    tuple val(meta), path(vcfs), path(tbis)
+    tuple val(meta), path(vcfs), path(tbis), path(bed)
```

Comment:
```suggestion
    tuple val(meta), path(vcfs), path(tbis)
    path(bed)
```

Reply:
You're merging mutliple vcf's here. I you want to limit the merge for _all_  vcf's to a region, it makes no sense to attach the bed to a single sample

Reply:
pinging @adamrtalbot for this. That was my first choice too, but here you give just one bed file for all the vcfs and tbis entered. But it's specific for these samples together and as @adamrtalbot said here: https://github.com/nf-core/modules/pull/1908#discussion_r924597652

Reply:
Okay, thanks for all the feedback!